### ISSUE TITLE
Respect ServicePointManager.DefaultConnectionLimit for .NET Framework in parallel operations

### ISSUE THREAD

**Issue by JoshLove-msft** [Azure/azure-sdk-for-net#9087]
For parallel upload and download, we use a default of 5 connections. For .NET Framework, the default value is 2.
We should consider whether we should use Min(5, DefaultConnectionLimit) for .NET Framework. For .NET core there is no maximum - https://blogs.msdn.microsoft.com/timomta/2017/10/23/controlling-the-number-of-outgoing-connections-from-httpclient-net-core-or-full-framework/

Note that we updated StorageTestBase to use more connections for our live tests. This issue is suggesting that we investigate whether this should be done for product code as well.

**Comment by JimSuplizio** [comment]
Hi @JoshLove-msft, we deeply appreciate your input into this project. Regrettably, this issue has remained inactive for over 2 years, leading us to the decision to close it. We've implemented this policy to maintain the relevance of our issue queue and facilitate easier navigation for new contributors. If you still believe this topic requires attention, please feel free to create a new issue, referencing this one. Thank you for your understanding and ongoing support.
